app/views/games/index.html.erb

```diff
@@ -1,182 +1,51 @@
 <div class="container-fluid">
-  <section id="register">
-    <div class= "row">
+  <section id="games-list">
+    <div class="row">
       <div class="col-sm-5 col-sm-offset-1">
         <%= image_tag 'chess.jpg', class: 'img-responsive feature-pic' %>
       </div>
-      <div class="col-sm-3 col-sm-offset-1 new-account">
-      <%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
-        <%= f.error_notification %>
-        <div class="form-group">
-          <%= f.email_field :email, class: 'form-control', required: true, autofocus: true, placeholder: 'Email Address' %>
-        </div>
-        <br />
-        <div class="form-group">
-          <%= f.password_field :password, class: 'form-control', required: true, hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length), placeholder: 'Password' %>
-        </div>
-        <br />
-        <div class="form-group">
-          <%= f.password_field :password_confirmation, class: 'form-control', required: true, placeholder: 'Confirm Password' %>
-        </div>
-        <br />
-        <div class="form-group">
-          <%= f.button :submit, "Create An Account", class: "btn btn-success form-control" %>
-        </div>
-      <% end %>
+      <div class="col-sm-5">
+        <h2 class="join-headline text-center">Your opponent awaits you! Join the battle!</h2>
+          <h4>Available game<a class="btn btn-success pull-right" href="#" role="button">Join game!</a></h4>
+          <br class="clr" />
```

Comment:
Even though you see them in the Rails scaffolds all the time, `<br />` is bad practice because it is not [semantic](https://en.wikipedia.org/wiki/Semantic_Web). They're basically only valid when outputting data that explicitly needed a line break, but still it's better to use `<pre>` in those instances.

In order to guarantee that objects are in different visual blocks in all browsers, you should make sure they are in different logical blocks (aka utilize encapsulation to provide your visual looks, similar to the way bootstrap does - think Russian nesting doll).

In most cases, the direct replacement for a `<br/>` is `<p></p>`